packages/core/src/components/TextField/TextField.tsx

```diff
@@ -306,7 +312,7 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
               onFocus={onFocus}
               onKeyDown={onKeyDown}
               onWheel={onWheel}
-              maxLength={maxLength}
+              maxLength={maxLength && !allowExceedingLimit ? maxLength : undefined}
```

Comment:
While using `allowExceedingLimit`, how would screen readers figure out the character count?

Reply:
I could add a `aria-describedby` field that reads out a hidden element with something along the lines of  `Maximum of X characters.`. The element wouldn't exist if `maxLength` will be set. Thoughts?

I personally haven't worked much with this type of accessibility so any inputs welcome :)

Reply:
Sounds good to me, we also have a HiddenText component for that